libs/sql-schema-describer/src/sqlite.rs

```diff
@@ -52,6 +81,22 @@ impl SqlSchemaDescriber {
         names
     }
 
+    fn get_size(&self, _schema: &str) -> usize {
+        debug!("Getting db size");
+        let sql = format!(r#"SELECT page_count * page_size as size FROM pragma_page_count(), pragma_page_size();"#);
+        let result = self.conn.query_raw(&sql, &[]).expect("get db size ");
+        let size: i64 = result
+            .first()
+            .map(|row| {
+                row.get("size")
+                    .and_then(|x| x.as_i64())
+                    .expect("convert db size result")
+            })
```

Comment:
It's definitely not important at all, but this can be flattened too with `and_then`